docs/deploy-targets.md

```
@@ -0,0 +1,443 @@
+)
+
+# EU cluster
+eu_cluster = DeployTarget("eu-west",
+    DeployTargetArgs(
+        name="eu-west-cluster",
+        kubernetes_endpoint="https://k8s-eu.example.com:6443",
+        kubernetes_token="<eu-token>",
+        router_pattern="${project}.${environment}.eu.example.com",
+    )
+)
+
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)